src/ddsrt/src/ifaddrs/windows/ifaddrs.c

```diff
@@ -242,7 +242,7 @@ copyaddr(
         }
       }
 
-      assert(i < addrtable->dwNumEntries);
+      //assert(i < addrtable->dwNumEntries);
```

Comment:
In Windows, this assertion triggers if the interface has no IP address assigned to it. If the for example the following command is executed:
`netsh interface ip set address name="Ethernet 4" static 172.19.5.2/24`
It first deletes the address before it adds the new one. Hence there is a short moment when the network interface has no IP assigned.